Source/Makefile

```diff
@@ -12,7 +12,7 @@ $(GPGMAILS): Libmacgpg
 	cp -r Libmacgpg/build/Release/Libmacgpg.framework $@/Frameworks
 	$(MAKE) -C $@
 	mkdir -p bundles/Free-$@.mailbundle
-	cp -r $@/build/Release/GPGMail.mailbundle bundles/Free-$@.mailbundle/Free-$@.mailbundle
+	cp -r $@/build/Release/GPGMail.mailbundle bundles/Free-$@.mailbundle
```

Comment:
No, this is intentional. Note that there is no trailing `/`. The result is not an additional directory level but a rename of the copied directory.

Without your change, the produced archive has a toplevel directory of `Free-GPGMail_X.mailbundle`. With your change, it contains only `GPGMail.mailbundle` making it not distinguishable from the non-free package.

Please revert the commit (or reset HEAD to the last one)

Reply:
So help me understand then, because when I build it, this is what I see: 

```
❮master↓1❯ jpr5@beast(Source)⇒ ls bundles/Free-GPGMail_4.mailbundle/Free-GPGMail_4.mailbundle   
Contents/
```

So, I go to `bundles/` and grab the `.mailbundle` from there, drop it into `~/Library/Mail/Bundles/`, which then leaves me with `~/Library/Mail/Bundles/Free-GPGMail_4.mailbundle`, which instead of containing a `Contents/` directory, itself contains another `Free-GPGMail_4.mailbundle/` (which itself then has the `Contents/` directory).  

This doesn't seem correct to me at all.  Can you explain to me why you think it is?

Reply:
The reason is the Github actions workflow.

https://github.com/bnavigator/Free-GPGMail/blob/ea47ee52660bf818ceb067bd31e14c252403e06d/.github/workflows/make.yml#L24-L35

This takes the **contents of** `Source/bundles/Free-GPGMail_3.mailbundle/` and provides them as artifact, which I can use for  the https://github.com/bnavigator/Free-GPGMail/releases pages.

With your change, the artifact from your PR contains the not renamed GPGMail.mailbundle as toplevel dir:
```
greiner@greinerT450s:~/ > unzip -l Free-GPGMail_4.mailbundle.zip 
Archive:  Free-GPGMail_4.mailbundle.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2020-11-19 00:25   GPGMail.mailbundle/
        0  2020-11-19 00:25   GPGMail.mailbundle/Contents/
        0  2020-11-19 00:25   GPGMail.mailbundle/Contents/Frameworks/
```

But it should look like:

```
greiner@greinerT450s:~/ > unzip -l Free-GPGMail_4.mailbundle-2020.1.zip 
Archive:  Free-GPGMail_4.mailbundle-2020.1.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2020-07-15 21:45   Free-GPGMail_4.mailbundle/
        0  2020-07-15 21:45   Free-GPGMail_4.mailbundle/Contents/
```

Reply:
Ahh I see, so it's a question of which "use case" is more likely when visiting here -- downloading source, compiling by hand and manually copying the bundle (which is me) or skipping all that and just downloading the available bin.  

In my case, what the automation does _isn't_ Right (because a bundle nested inside a bundle makes no sense), but I definitely see that for the GitHub Workflow it _is_ Right (so that the .zip is created correctly).

Is there a compromise here where both use cases work?  Maybe an extra step that copies the properly-structured bundle to a "github package build" dir with an extra level of nesting so it works correctly?

Reply:
Yes, let me open a new pull request regarding this, where we can test things out. Please revert here as it is unrelated to the Big Sur updated and I will merge your PR

Reply:
#6